jobs/scripts/gluster-integration/gluster-integration.sh

```diff
@@ -36,12 +37,14 @@ cd "${GIT_REPO_NAME}"
 if [ "${GIT_TARGET_REPO}" = "sit-test-cases" ]; then
 	if [ -n "${ghprbPullId}" ]; then
 		# Just invoke "make test" with the corresponding parameters.
-		TEST_EXTRA_VARS="test_repo=${GIT_TARGET_REPO_URL} test_repo_pr=${ghprbPullId}"
+		TEST_EXTRA_VARS="test_repo=${GIT_TARGET_REPO_URL} \
+					test_repo_pr=${ghprbPullId} \
+					backend=${BACKEND}"
 	fi
 else
 	if [ -n "${ghprbPullId}" ]; then
 		# Run sanity tests only for pull requests on sit-environment
-		TEST_EXTRA_VARS="test_sanity_only=1"
+		TEST_EXTRA_VARS="test_sanity_only=1 backend=${FILE_SYSTEM}"
```

Comment:
should this be ${BACKEND}?

Reply:
Ah..my bad. It should still work but for completeness we could use `${BACKEND}`.